packages/web-components/src/components/ic-stepper/ic-stepper.stories.mdx

```diff
@@ -95,3 +96,169 @@ import Step from "../ic-step/readme.md";
     </ic-stepper>`}
   </Story>
 </Canvas>
+
+### Compact stepper
+
+<Canvas>
+  <Story name="Compact stepper" parameters={{ loki: { skip: true } }}>
+    {html`<ic-stepper variant="compact" id="standard-compact-stepper">
+        <ic-step step-title="Create" step-type="completed"></ic-step>
+        <ic-step step-title="Read" step-status="complete" current></ic-step>
+        <ic-step step-title="Update" step-status="not required"></ic-step>
+        <ic-step step-title="Delete" step-status="not required"></ic-step>
+      </ic-stepper>
+      <ic-button
+        style="margin-right: var(--ic-space-lg)"
+        onclick="toPreviousStep()"
+        >Back</ic-button
+      >
+      <ic-button onclick="toNextStep()">Next</ic-button>
+      <script>
```

Comment:
This doesn't seem to be working - looks like it's setting the 'current' prop on the visible step and the loading indicator isn't showing 
![image](https://user-images.githubusercontent.com/111882109/208152387-90a2dc17-15e0-413a-8121-aa9d202437c9.png)
🙁 or is this the [bug you mentioned that you're working on](https://github.com/mi6/ic-ui-kit/pull/182#issuecomment-1352983292)?

Reply:
I am currently looking into the best way to show current steps in the compact stepper. Step type current overwrites any step type the step had before it was the step in view